upup/models/cloudup/_aws/topologies/_topology_private/network.yaml

```diff
@@ -190,6 +190,13 @@ loadBalancer/api.{{ ClusterName }}:
   {{ end }}
   listeners:
     443: { instancePort: 443 }
+  scheme: internal
```

Comment:
Yes, can we please remove this and we can cover the feature in another PR? For now ELB's are our main entry point into the network, so we need them exposed. Also this PR shouldn't change functionality, just map it to Terraform is all